### ISSUE TITLE
N1QL can't parse a query with multiple joins 

### ISSUE THREAD

**Issue by stephenlang84** [couchbase/couchbase-lite-core#1107]
Hi Team.

I'm trying the multiple joins by N1SQL query, but I got a parsing error
```
"SELECT rec, dss, dem FROM db rec LEFT JOIN db dss ON rec.sessionId = dss.meta.id LEFT JOIN db dem ON rec.demId = dem.meta.id WHERE rec.meta.id LIKE 'rec:%'"

ASSERT failure in c4query_new: "N1QL syntax error near character 82���������������", file D:\WORK\develop-new-cmake\cortex-v2\src\Cortex\CouchBaseNative\CouchQueryNative.cpp, line 38
```

But the single join works well.
```
"SELECT rec, dss FROM db rec LEFT JOIN db dss ON rec.sessionId = dss.meta.id WHERE rec.meta.id LIKE 'rec:%'"
```
Any suggestion ?

Thank you.

**Comment by stephlang** [comment]
I found an example from your unit test at https://github.com/couchbase/couchbase-lite-core/blob/4bebac69e5a2f8994da139c696ee8113cf3264b0/LiteCore/tests/N1QLParserTest.cc
Can you add more unit test for multiple joins ?

**Comment by jianminzhao** [comment]
The issue is migrated to https://issues.couchbase.com/browse/CBL-1560.

**Comment by stephlang** [comment]
> The issue is migrated to https://issues.couchbase.com/browse/CBL-1560.

Thanks you @jianminzhao

**Comment by borrrden** [comment]
The fix for this was included in 3.0.0.
